### PR TITLE
feat(dds/user): add new database user sdk support

### PR DESCRIPTION
**Pull request by Lance52259**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format,
  will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: openstack/dds/v3/users/requests.go [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+	Password string `json:"user_pwd" required:"true"`
+	// User name. Defaults to "rwuser".
+	// The length is 1~64 bits and can contain letters, numbers, hyphens, underscores and dots.
+	Name string `json:"user_name,omitempty"`
+	// The name of the database where the user is located.
+	// The length is 1~64 bits and can contain letters, numbers and underscores.
+	DbName string `json:"db_name,omitempty"`
+}
+
+// Update is a method to create a new database user using given parameters.
```
Comment (ShiChangkuo) [inline openstack/dds/v3/users/requests.go:113]: the function description is not correct

File: openstack/dds/v3/users/requests.go [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+func Update(c *golangsdk.ServiceClient, instanceId string, opts UpdateOpts) error {
```
Comment (ShiChangkuo) [inline openstack/dds/v3/users/requests.go:114]: suggest to rename as ResetPassword

Comment (Lance52259) [reply openstack/dds/v3/users/requests.go]: done.

File: openstack/dds/v3/users/requests.go (hunk 1 above)
Comment (Lance52259) [reply openstack/dds/v3/users/requests.go]: done.

**Pull request merged**